src/components/narrative/index.js

```diff
@@ -136,7 +126,26 @@ class Narrative extends React.Component {
     );
   }
   renderBlocks() {
-    const ret = this.props.blocks.map((b, i) => {
+    return this.props.blocks.map((b, i) => {
+
+      if (b.isEndOfNarrativeSlide) {
+        return (
+          <EndOfNarrative key="EON" id="EndOfNarrative">
```

Comment:
Does this need to get wrapped in a translate function for non-english narratives?

Reply:
It should I think, but this isn't a regression. There are a few strings in auspice which do not have translate calls, and which therefore haven't been added to the translation JSONs. Would be a good issue to make to track them all down.